src/govuk/components/summary-list/summary-list.yaml

```diff
@@ -787,3 +954,52 @@ examples:
           value:
             text: Firstname Lastname
           classes: app-custom-class
+  - name: summary card with custom classes
+    hidden: true
+    data:
+      card:
+        classes: custom-class
+      rows:
+        - key:
+            text: Name
+          value:
+            text: Firstname Lastname
+        - key:
+            text: Date of birth
+          value:
+            text: 13/08/1980
+  - name: summary card with custom attributes
+    hidden: true
+    data:
+      card:
+        attributes:
+          data-attribute-1: value-1
+          data-attribute-2: value-2
+      rows:
+        - key:
+            text: Name
+          value:
+            text: Firstname Lastname
+        - key:
+            text: Date of birth
+          value:
+            text: 13/08/1980
+  - name: summary card with only 1 action
+    hidden: true
+    data:
+      card:
+        title:
+          text: Undergraduate teaching assistant
+        actions:
+          items:
+            - text: My lonely action
+              href: '#'
+      rows:
+        - key:
+            text: Name
+          value:
+            text: Firstname Lastname
+        - key:
+            text: Date of birth
+          value:
+            text: 13/08/1980
```

Comment:
Sorry @owenatgov mind sticking a trailing new line here?

Saves the next person accidentally inserting a new line on save. If you fancy adding a plugin for [EditorConfig](https://editorconfig.org/#download) the project `.editorconfig` config will fix it for you